### ISSUE TITLE
UIWebView APIs Deprecated

### ISSUE THREAD

**Issue by jamesjara** [wix/react-native-ui-lib#718]
Very worried about this, it this under the near roadmap? 

```
ITMS-90809: Deprecated API Usage - Apple will stop accepting submissions of new apps that use UIWebView APIs starting from April 2020. See https://developer.apple.com/documentation/uikit/uiwebview for more information.
```

https://github.com/wix/react-native-ui-lib/blob/f0a2de6c6124e28b1f53784b8819e91447208804/lib/ios/reactnativeuilib/keyboardtrackingview/KeyboardTrackingViewTempManager.m

**Comment by ethanshar** [comment]
Yes, since we depend on the that component as well we plan to migrate from UIWebView.

**Comment by cosmarc** [comment]
@ethanshar any news on this?

**Comment by ethanshar** [comment]
Hey @cosmarc 
Unfortunately no. we're missing an iOS developer at the moment. 
If this is something you can PR us with please go ahead.

**Comment by Macrow** [comment]
I receive an email from Apple:

```
ITMS-90809: Deprecated API Usage - New apps that use UIWebView are no longer accepted. Instead, use WKWebView for improved security and reliability. Learn more (https://developer.apple.com/documentation/uikit/uiwebview).
```
then I found this:
```
./node_modules/react-native-ui-lib/lib/ios/reactnativeuilib/keyboardtrackingview/KeyboardTrackingViewTempManager.m:-(void)_swizzleWebViewInputAccessory:(UIWebView*)webview
./node_modules/react-native-ui-lib/lib/ios/reactnativeuilib/keyboardtrackingview/KeyboardTrackingViewTempManager.m:        else if ([subview isKindOfClass:[UIWebView class]])
./node_modules/react-native-ui-lib/lib/ios/reactnativeuilib/keyboardtrackingview/KeyboardTrackingViewTempManager.m:            [self _swizzleWebViewInputAccessory:(UIWebView*)subview];
```

I'm eager to know how to fix this...

**Comment by ethanshar** [comment]
Hey, 
This is something we need to internally migrate in our components and planning to do in the upcoming week
Stay tune

**Comment by ethanshar** [comment]
Fixed in 5.7.2
